open-ce/build_env.py

```diff
@@ -123,7 +123,7 @@ def build_env(arg_strings=None):
 
         conda_env_data.update_conda_env_file_content(build_command, build_tree)
 
-    conda_env_files = conda_env_data.write_conda_env_files()
+    conda_env_files = conda_env_data.write_conda_env_files(args.output_folder)
```

Comment:
I think we should probably change the default argument of `write_conda_env_files` to DEFAULT_OUTPUT_FOLDER instead of os.getcwd().